plugins/webp-uploads/readme.txt

```diff
@@ -63,8 +63,17 @@ By default, the Modern Image Formats plugin will only generate WebP versions of
 == Changelog ==
 
 = 2.0.0 =
+
+**Features**
+
 * Add AVIF image format support. Add setting for output image format to choose between WebP and AVIF. ([1176](https://github.com/WordPress/performance/pull/1176))
 
+**Enhancements**
+
+* Add `picture` element support. ([73](https://github.com/WordPress/performance/pull/73))
```

Comment:
Consider moving picture element support to Features section?

Reply:
Done in 4529501dcd8b459dda68e3aa2e125f249ee5d310